pkg/openstackclient/funcs.go

```diff
@@ -50,6 +50,16 @@ func ClientPodSpec(
 		envVars["PROMETHEUS_CA_CERT"] = env.SetValue(tls.DownstreamTLSCABundlePath)
 	}
 
+	if instance.Spec.Env != nil {
+		for idx := range instance.Spec.Env {
+			e := instance.Spec.Env[idx]
+			envVars[e.Name] = func(env *corev1.EnvVar) {
```

Comment:
this will override the existing env vars, do we want that?

Reply:
it was on purpose doing it like that, as it would be the same behavior as we have with the different overrides we have.